public/blog/crosswalk-app-tools-alpha.md

```diff
@@ -0,0 +1,70 @@
+We are happy to announce the alpha version for Crosswalk-app-tools. This is a preview milestone of our forthcoming Node.js based tools for Android app development. Deployment in a production environment is not encouraged at this time.
+
+Crosswalk-app-tools is licensed under the Apache License Version 2.0, please refer to the file LICENSE-APACHE-V2 included in the package.
+
+### Installation
+
+Prerequisites are functional
+
+  1. Android SDK
+  2. Apache Ant
+  3. Node.js
+
+on the system.
+
```

Comment:
This sounds awkward to me:
" Prerequisites are functional
    Android SDK
    Apache Ant
    Node.js
on the system. "
How about:
" The following components are required:
    Android SDK     (https://developer.android.com/sdk/index.html)
    Apache Ant  (http://ant.apache.org/bindownload.cgi)
    Node.js  (https://nodejs.org/)
" 
...and each line is a link to the best known download location.   Is there any required minimal version for any, or have we tested with any specific version?   (e.g. node.js > 0.10 ? )